src/shared/components/home/instances.tsx

```diff
@@ -148,9 +149,15 @@ export class Instances extends Component<any, InstancesState> {
             {items.map(i => (
               <tr key={i.domain}>
                 <td>
-                  <a href={`https://${i.domain}`} rel={relTags}>
-                    {i.domain}
-                  </a>
+                  {link === true || link === undefined ? (
```

Comment:
Once you merge the other suggestion, this can become:
```suggestion
                  {link ? (
```